packages/blockchain-extension/extension/commands/gatewayConnectCommand.ts

```diff
@@ -124,15 +123,14 @@ export async function gatewayConnect(gatewayRegistryEntry: FabricGatewayRegistry
 
     const connectionProfilePath: string = await FabricGatewayHelper.getConnectionProfilePath(gatewayRegistryEntry.name);
 
-    const connection: IFabricClientConnection = FabricConnectionFactory.createFabricClientConnection(connectionProfilePath);
+    const connection: IFabricGatewayConnection = FabricConnectionFactory.createFabricGatewayConnection(connectionProfilePath);
```

Comment:
Similar to my last comment - maybe rename FabricConnectionFactory to FabricGatewayFactory?

Reply:
that handles the environment connection too so doesn't need to change

Reply:
Ignore this - just noticed we have FabricEnvironmentConnections in this class!

Reply:
Beat me to it by 16 seconds!